src/explorer-link.ts

```diff
@@ -1,9 +1,16 @@
+import { addPathToUrl } from './helpers';
 import prefixForChain from './prefix-for-chain';
-import prefixForNetwork from './prefix-for-network'
+import prefixForNetwork from './prefix-for-network';
+     
+export function createCustomExplorerLink(hash: string, customNetworkUrl: string): string {
+  let parsedUrl = addPathToUrl(customNetworkUrl, 'tx', hash) 
+    return parsedUrl;
+  }
```

Comment:
Nit: Inconsistent indentation, and use of `let` over `const`:
```suggestion
  const parsedUrl = addPathToUrl(customNetworkUrl, 'tx', hash) 
  return parsedUrl
}
```